activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+* Updated `parameterize` to use keyword arguments and added the ability to preserve case of parameterized string.
```

Comment:
That the method uses keyword arguments is more of an implementation detail that I don't think we should mention, can you take that out? :smile: